lib/MusicBrainz/Server/Data/Relationship.pm

```diff
@@ -823,8 +825,18 @@ sub exists {
 
     $self->_check_types($type0, $type1);
 
-    my @props = qw(entity0 entity1 link_order link);
-    my @values = @{$values}{qw(entity0_id entity1_id link_order)};
+    my @props = qw(entity0 entity1);
+    my @values = @{$values}{qw(entity0_id entity1_id)};
+
+    # For part of series, do not count link order as different
+    # if everything else is the same in order to avoid duplicate additions
+    # (different number attributes still allowed)
+    my $ignore_link_order = grep { $_ eq $values->{link_type_id} } values %PART_OF_SERIES_ROW_IDS;
```

Comment:
Should we just store them as a list like `PART_OF_SERIES_LINK_TYPE_IDS` in root/static/scripts/common/constants.js so that we don't need to convert it via `values`? (At least until we have a use for looking them up by entity type.) I guess we could also use `contains_number` from `Data::Utils`, then. :)

Reply:
Oh, I didn't realize we were already doing it that way in JS. Then we might as well, sure.